tests/test_octodns_provider_dnsimple.py

```diff
@@ -38,7 +38,13 @@ class TestDnsimpleProvider(TestCase):
             break
 
     def test_populate(self):
+
+        # Sandbox
+        provider = DnsimpleProvider('test', 'token', 42, 'true')
+        self.assertTrue('sandbox' in provider._client.base)
+
         provider = DnsimpleProvider('test', 'token', 42)
+        self.assertFalse('sandbox' in provider._client.base)
```

Comment:
Realized this was a good idea and just added it rather than doing another round trip.